src/Lumina/Data/Parsing/MdlStructs.cs

```diff
@@ -337,15 +337,15 @@ public static ShapeStruct Read( LuminaBinaryReader br )
 
         public struct ShapeMeshStruct
         {
-            public uint StartIndex;
+            public uint MeshIndexOffset; // from TT: The offset to the index block this Shape Data should be replacing in. -- This is how Shape Data is tied to each mesh.
             public uint ShapeValueCount;
             public uint ShapeValueOffset;
         }
 
         public struct ShapeValueStruct
         {
-            public ushort Offset;
-            public ushort Value;
+            public ushort BaseIndicesIndex; // PROBABLY: Index into the Indices array of a mesh.
+            public ushort ReplacingVertexIndex; // PROBABLY: Index into the (without transformation probably unused) vertex of a mesh.
```

Comment:
there probs should be obsolete backing fields for this but i dont know if i care